app/assets/stylesheets/partials/_content_page.scss

```diff
@@ -0,0 +1,27 @@
+  .content {
+    max-width: 33em;
+    margin-left: 40em;
+    @include media(medium) {
```

Comment:
I think we might need an additional larger breakpoint than medium. The text is still crammed up against the right edge by the time this breakpoint kicks in.

Reply:
Looks better with the other change I made.